docs/fundamentals/connection/connect-to-mongodb.txt

```diff
@@ -0,0 +1,300 @@
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to connect your Laravel application to a
+MongoDB instance or replica set deployment by using {+odm-short+}.
+
+This guide includes the following sections:
+
+- :ref:`Connection URI <laravel-connection-uri>`, which explains connection
+  URIs and their constituent parts
+- :ref:`laravel-database-config`, which explains how to set up your MongoDB
+  database connection for your Laravel app.
+- :ref:`Connection Example <laravel-atlas-connection-example>`, which  provides
+  examples that show how to connect to MongoDB by using an Atlas connection
+  string.
+- :ref:`laravel-other-ways-to-connect` describes ways to connect to MongoDB
+  deployments that are not hosted on Atlas.
+
+.. _laravel-connection-uri:
+
+Connection URI
+--------------
+
+A **connection URI**, also known as a connection string, specifies how
+{+odm-short+} connects to MongoDB and how to behave while connected.
+
+Parts of a Connection URI
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The following figure explains each part of a sample connection URI:
+
+.. figure:: /includes/figures/connection_uri_parts.png
+   :alt: Parts of a connection URI
+
+In this connection URI, ``mongodb+srv`` is the protocol, which uses the
+:manual:`DNS Seed List Connection Format </reference/connection-string/#dns-seed-list-connection-format>`
+for greater flexibility in your deployment and the ability to change the
+servers in rotation without reconfiguring clients.
+
+If the machine that hosts your MongoDB deployment does not support this
+feature, use protocol for the
+:manual:`Standard Connection String Format </reference/connection-string/#std-label-connections-standard-connection-string-format>`
+instead.
+
+If you use a password-based authentication, the part of the connection
+string after the protocol contains your username and password. Replace the
+placeholder for ``user`` with your username and ``pass`` with your password.
+If you use an authentication mechanism that does not require a username
+and password, omit this part of the connection URI.
+
+The part of the connection string after the credentials specifies your MongoDB
+instance's hostname or IP address and port. The preceding example uses
+``sample.host`` as the hostname and ``27017`` as the port. Replace these values
+to point to your MongoDB instance.
+
+The last part of the connection string specifies connection and authentication
+options. In the example, we set the following connection options and values:
+
+- ``maxPoolSize=20``
+- ``w=majority``
+
+.. _laravel-database-config:
+
+Laravel Database Connection Configuration
+------------------------------------------
+
+{+odm-short+} lets you configure your MongoDB database connection in the
+``config/database.php`` Laravel application file. You can specify the following
+connection details in this file:
+
+- ``default``, which specifies the default database connection to use when
+  unspecified
```

Comment:
```suggestion
- ``default``, which specifies the default database connection to use
```

Reply:
Thanks for the suggestion. I wanted to emphasize the behavior of the default value, but now think maybe including "default" in the description of itself might be confusing. think maybe a more clear way to describe this could be:
"default, which specifies the database connection to use when unspecified"